includes/class-wc-payments-order-service.php

```diff
@@ -314,15 +314,17 @@ public function mark_order_blocked_for_fraud( $order, $intent_id, $intent_status
 	 * @param string   $amount     The disputed amount – formatted currency value.
 	 * @param string   $reason     The reason for the dispute – human-readable text.
 	 * @param string   $due_by     The deadline for responding to the dispute - formatted date string.
+	 * @param string   $status     The status of the dispute.
 	 *
 	 * @return void
 	 */
-	public function mark_payment_dispute_created( $order, $charge_id, $amount, $reason, $due_by ) {
+	public function mark_payment_dispute_created( $order, $charge_id, $amount, $reason, $due_by, $status = '' ) {
 		if ( ! is_a( $order, 'WC_Order' ) ) {
 			return;
 		}
 
-		$note = $this->generate_dispute_created_note( $charge_id, $amount, $reason, $due_by );
+		$is_inquiry = strpos( $status, 'warning' ) === 0;
```

Comment:
Nitpick ⛏️  - should we change 'warning' to 'warning_'  to make it similar to line 351?
```suggestion
		$is_inquiry = strpos( $status, 'warning_' ) === 0;
```